frontend/packages/ceph-storage-plugin/integration-tests-cypress/support/index.ts

```diff
@@ -1,55 +1,107 @@
 import '../../../integration-tests-cypress/support/index.ts';
-import { OCS_OP } from '../consts';
+import { ODF_OP, STORAGE_SYSTEM_NAME, OCS_SC_STATE } from '../consts';
 import { CATALOG } from '../mocks/install';
 import { NS } from '../utils/consts';
 import {
+  patchOperatorHubConfig,
   createImagePullSecret,
   createCustomCatalogSource,
   subscribeToOperator,
   linkPullSecretToPods,
-  createInternalStorageCluster,
-  verifyMonitoring,
-  verifyNodeLabels,
-  verifyClusterReadiness,
 } from '../views/install';
 
 declare global {
   namespace Cypress {
     interface Chainable<Subject> {
-      install(encrypted?: boolean): Chainable<Element>;
+      install(): Chainable<Element>;
     }
   }
 }
 
-Cypress.Commands.add('install', (encrypted = false) => {
-  cy.exec(`oc get storagecluster ocs-storagecluster -n ${NS}`, {
+const installODFOperator = () => {
+  cy.log('Perform ODF Operator installation and StoargeSystem creation');
+  patchOperatorHubConfig();
+  createImagePullSecret(CATALOG.NAMESPACE);
+  createCustomCatalogSource();
+  subscribeToOperator();
+  cy.byTestID('view-installed-operators-btn').click();
+  cy.log('Wait for operator phase to be `Installing`');
+  cy.byLegacyTestID('item-filter').type(`${ODF_OP}`);
+  cy.byTestID('status-text', { timeout: 120000 }).should('have.text', 'Installing');
+  // Waiting for sometime to make sure service accounts are loaded by the operator.
+  // eslint-disable-next-line cypress/no-unnecessary-waiting
+  cy.wait(30000);
+  linkPullSecretToPods();
+  // eslint-disable-next-line cypress/no-unnecessary-waiting
+  cy.wait(30000);
+  linkPullSecretToPods(false);
+  // eslint-disable-next-line cypress/no-unnecessary-waiting
+  cy.wait(120000);
+  linkPullSecretToPods(false);
+  // eslint-disable-next-line cypress/no-unnecessary-waiting
+  cy.wait(120000);
```

Comment:
Too many arbitrary waits. I would rather we poll on something.

Reply:
I copied from here: https://github.com/red-hat-storage/odf-console/blob/9cd8ecd5a50023d0058165574ae4591680d549b1/test-prow-e2e.sh#L90

Earlier as well we were waiting: https://github.com/openshift/console/blob/108daa3c51158e6fef665da1834d6d7a933f9e18/frontend/packages/ceph-storage-plugin/integration-tests-cypress/support/index.ts#L38 I just added few more wait as per odf-console's script.